### DISCUSSION TITLE
Pool not synching on 2nd farm (Win GUI 1.1.784)

### DISCUSSION THREAD

**Discussion by ASPANSolutions** [Chia-Network/chia-blockchain#7102] [Support]
Pool and plotnft not updating via wallet and blockchain sync.  The 2nd farm is blank of all pool data.  Both machines at same block height and all is working except for the lack of Pool information on 2nd farm.  I believe this is unique to the windows GUI.

The keys are the same. Wallet address, balances etc match. On first wallet shows block 50948 transaction is outbound (e.g. setting up pool) value 0.000000000001, and on 2nd farm shows as incoming value 0.000000999999. All other transactions in the wallet match between the two nodes.

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
